### PR TITLE
DOP-4296: add portuguese and korean to language selector

### PR DESCRIPTION
**Pull request by caesarbell**
### Stories/Links:

DOP-4296

### Current Behavior:

[Production](https://www.mongodb.com/docs/drivers/)
[DOP-4295 Stage Link](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/e019b9cda6d0ff2f978b2fc42c2fc14b0d2af9bc/master/drivers/caesarbell/DOP-4295/index.html)

### Staging Links:

[Staged Link](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/2dd5b5e744c862a7fd57502feb5579a59eaaac80/master/docs/caesarbell/DOP-4296/index.html)

### Notes:

This PR adds two additional languages to the language flipper, i.e. Korean and Portuguese.

### CODE REVIEW

File: src/components/DocumentBody.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const getAnonymousFootnoteReferences = (index, numAnonRefs) => {
 };
 
 const HIDE_UNIFIED_FOOTER_LOCALE = process.env['GATSBY_HIDE_UNIFIED_FOOTER_LOCALE'] === 'true';
-const AVAILABLE_LANGUAGES = ['English', '简体中文'];
+const AVAILABLE_LANGUAGES = ['English', '简体中文', '한국어', 'Português'];
 
 const DocumentBody = (props) => {
   const {
```
Comment (schmalliso) [inline src/components/DocumentBody.js:76]: Comment for line 87 - can we clean up that comment, please :)

File: src/components/DocumentBody.js [hunk 2]
```diff
@@ -121,8 +121,10 @@ const DocumentBody = (props) => {
   const slugForUrl = slug === '/' ? `${withPrefix('')}` : `${withPrefix(slug)}`; // handle the `/` path
   const onSelectLocale = (locale) => {
     const localeHrefMap = {
-      'zh-cn': `https://mongodbcom-cdn.staging.corp.mongodb.com/zh-cn${slugForUrl}/`,
-      'en-us': `https://mongodbcom-cdn.website.staging.corp.mongodb.com${slugForUrl}/`,
+      'zh-cn': `${location.origin}/zh-cn${slugForUrl}`,
```
Comment (seungpark) [inline src/components/DocumentBody.js:124]: we can use the util function `assertTrailingSlash` to validate trailing slashes for parts / slug here

**Review by seungpark — COMMENTED** [review]
to verify, are there any stage sites that include the language flipped?

**Comment by caesarbell** [comment]
> to verify, are there any stage sites that include the language flipped?

I am working on that now. Before moving this from Draft, there will be a staging link showcasing the language flipper.

**Comment by caesarbell** [comment]
> Staged Link

Just to confirm you are not talking about the stage commits correct, we are talking about seeing this on a `pre-prd` env?

**Comment by seungpark** [comment]
> > Staged Link
> 
> Just to confirm you are not talking about the stage commits correct, we are talking about seeing this on a `pre-prd` env?

yup. just wanted to confirm this is the correct path (in diff environments). i think we had a view of this before?

**Comment by caesarbell** [comment]
> > > Staged Link
> > 
> > 
> > Just to confirm you are not talking about the stage commits correct, we are talking about seeing this on a `pre-prd` env?
> 
> yup. just wanted to confirm this is the correct path (in diff environments). i think we had a view of this before?

We can view it [here](https://mongodbcom-cdn.website.staging.corp.mongodb.com/docs-qa/atlas/master/) for the work done in DOP-4295

**Comment by caesarbell** [comment]
We now have the `pre-prd` [deployed](https://mongodbcom-cdn.website.staging.corp.mongodb.com/docs-qa/atlas/master/) with the changes from this PR

**Review by seungpark — APPROVED** [review]

**Pull request merged**